terraform/helm/vector-log-agent/testing/test.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
nice! took me a while to figure out what we're doing here, but iiuc we're only testing the vector transforms here. Can we rename this to `test_transform.sh` or something?